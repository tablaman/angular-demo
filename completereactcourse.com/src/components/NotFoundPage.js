import React from 'react'

const NotFound = () => (
   <h1>The Four Zero Four!</h1>
)

export default NotFound;