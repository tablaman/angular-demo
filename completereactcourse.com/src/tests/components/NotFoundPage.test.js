import React from 'react';
import {
  shallow
} from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';
import expenses from '../fixtures/expenses'

test('render ExpenseListItem with fixture data', () => {
  const wrapper = shallow( < NotFoundPage /> );
  expect(wrapper).toMatchSnapshot();
});