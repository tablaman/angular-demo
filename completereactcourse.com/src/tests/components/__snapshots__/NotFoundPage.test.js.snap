// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ExpenseListItem with fixture data 1`] = `
<h1>
  The Four Zero Four!
</h1>
`;
